### ISSUE TITLE
ezntfs-app not working on Catalina

### ISSUE THREAD

**Issue by BotJohn** [undecidabot/ezntfs#4]
Hello,

I believe that this is already a known issue, but I cannot install ezntfs-app on OS X Catalina.

`sudo ezntfs-app install` results in the following error:
```
>>> /private/etc/sudoers.d/com-lezgomatt-ezntfs: syntax error near line 1 <<<
sudo: parse error in /private/etc/sudoers.d/com-lezgomatt-ezntfs near line 1
sudo: no valid sudoers sources found, quitting
sudo: unable to initialize policy plugin
```
Are there any safe workarounds to this issue? (@theperspectiv 's fix involved a risk of breaking the OS)
Is the app expected to be patched for prior versions of macOS in the future, or is it strictly for OS 11+?

**Comment by lezgomatt** [comment]
Hi @BotJohn, thanks for the report. I don't see any reason why this wouldn't work on Catalina, I just can't test it myself.

Could show me the contents of your `/private/etc/sudoers.d/com-lezgomatt-ezntfs` file? Don't worry, there's nothing confidential there. You should be able to do this by doing the following steps:

1. Open finder and go to `/private/etc/sudoers.d/` (or run `open /etc/sudoers.d/` in the terminal).
2. Right click the file, and click "Get Info" (or select the file and click `⌘+i`).
3. Expand the "Sharing & Permissions" panel, and click the lock icon at the bottom right to unlock it.
4. Give everyone "Read & Write" access to the file (don't forget to revert this to "No Access" after copying its contents).
5. If your `sudo` command is broken, you might want to delete the file too.

**Comment by BotJohn** [comment]
> Could show me the contents of your `/private/etc/sudoers.d/com-lezgomatt-ezntfs` file? Don't worry, there's nothing confidential there. 

`%#20		ALL = NOPASSWD: /usr/local/bin/ntfs-3g`

Weirdly enough, changing the access allowed me to install ezntfs-app. However, I now get the same error whenever I try to use any sudo command - related to ezntfs or not - unless com-lezgomatt-ezntfs has read & write permissions for everyone.

```
>>> /private/etc/sudoers.d/com-lezgomatt-ezntfs: syntax error near line 1 <<<
sudo: parse error in /private/etc/sudoers.d/com-lezgomatt-ezntfs near line 1
sudo: no valid sudoers sources found, quitting
sudo: unable to initialize policy plugin
```
If permissions are kept open, then I get an annoying reminder that the file is fully writable
`/private/etc/sudoers.d/com-lezgomatt-ezntfs is world writable`

**Comment by lezgomatt** [comment]
Oh, I think I've figured it out. I think the file is missing a newline at the end (everything else looks fine).

Could you edit that file (grant yourself read & write access again), and add a newline at the end? I think the older versions of `sudo` are quite strict about the format.

After editing the file, try running `sudo --non-interactive /usr/local/bin/ntfs-3g` to confirm that the new config works.

Please do *not* re-run the `install` command while doing this, as doing so would reset the file's contents. I will update the install script once I receive confirmation from you that this works. Thanks!

**Comment by BotJohn** [comment]
@lezgomatt Unfortunately the issue still seems to persist. Also, that command results in:
```
ntfs-3g: No device is specified.

ntfs-3g 2017.3.23 external FUSE 29 - Third Generation NTFS Driver
		Configuration type 1, XATTRS are on, POSIX ACLS are off
```

**Comment by lezgomatt** [comment]
Hi again @BotJohn, what makes you say that the issue still persists? Did you encounter the syntax/parse error again? It looks like the command you ran was successful, otherwise it would have failed with either the syntax/parse error again or a `sudo: a password is required` message.

I forgot to mention that after editing the file, be sure to revoke access for "everyone" (but make sure the "system" and "wheel" still have access of course). Sorry about that.

Just to be clear, it should only resolve the syntax/parse error. I will still need to fix the install script in a new release before you will be able to set up and use the GUI.

**Comment by lezgomatt** [comment]
Oh, if you could also share the output for `sudo --version`, that would be super helpful, thanks 🙏 

You might want to update it to the latest version by the way, there was a vulnerability patched recently ([CVE-2021-3156](https://support.apple.com/en-us/HT212177)).

**Comment by lezgomatt** [comment]
Released [v1.1.1](https://pypi.org/project/ezntfs/1.1.1/) just now. Please try this version, and kindly close this issue if it fixes it. Thanks again for the report :D

**Comment by BotJohn** [comment]
Working great now! Thank you so much for taking the time to fix the issue, I appreciate it!
